main/OpenCover.sln

```diff
@@ -108,6 +110,7 @@ Global
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{B2CE418E-A5C8-4C46-9513-771414B3CA4C}.Debug|Any CPU.ActiveCfg = Debug|Win32
+		{B2CE418E-A5C8-4C46-9513-771414B3CA4C}.Debug|Any CPU.Build.0 = Debug|Win32

```

Comment:
We don't use `Any CPU` please remove these platforms - we should only have x64 and x86